src/Arcus.Messaging.ServiceBus.Core/Extensions/AzureClientFactoryBuilderExtensions.cs

```diff
@@ -29,6 +29,7 @@ public static class AzureClientFactoryBuilderExtensions
         /// <exception cref="ArgumentException">Thrown when the <paramref name="connectionStringSecretName"/> is blank.</exception>
         /// <exception cref="InvalidOperationException">Thrown when the Arcus secret store is not registered.</exception>
         /// <exception cref="SecretNotFoundException">Thrown when no Azure EventHubs connection string secret was found in the Arcus secret store.</exception>
+        [Obsolete("Will be removed in v3.0, please use Microsoft's built-in Azure SDK clients to register a " + nameof(ServiceBusClient) + " to remove the " + nameof(ISecretProvider) + " requirement")]
```

Comment:
I believe we can make the entire class obsolete ?

Reply:
Yes, probably in time, but the scope of this PR is only to deprecate Arcus.Security references.

Reply:
Oh, my mistake, thought that this was another extensions class, yes we can.